userManagement/views.py

```diff
@@ -12,12 +14,12 @@ def settings(request):
     if request.method == 'POST':
         p_form = PasswordChangeForm(request.user, request.POST)
         n_form = notificationUpdateForm(request.POST, instance=request.user)
-        if p_form.is_valid():
+        if p_form.is_valid() and 'changePassword' in request.POST:
```

Comment:
Are we checking that the original password was correct? Do we have  testing for that?

Reply:
Django form automatically does that